docs/glossary.rst

```diff
@@ -1,2 +1,58 @@
 Glossary
 ========
+
+.. glossary::
+
+    :class:`~pulpcore.plugin.models.Artifact`
+        A file that belongs to a :term:`content unit<content>`.
+
+    :class:`~pulpcore.plugin.models.Content`
+        The smallest that of data that is managed by Pulp. When singular, "content unit" should be
+        used. Content is added and removed to :term:`Repositories<repository>`, and can have multiple
+        :term:`artifacts<artifact>`. Each content unit has a :term:`type` (like .rpm or .deb) which
+        that is defined by a :term:`plugin`
+
+    content app
+        A `Django <https://docs.djangoproject.com>`_ app provided by :term:`pulpcore` that serves
+        :term:`content`.
+
+    :class:`~pulpcore.plugin.models.Distribution`
+        User facing settings that specify how and where associated
+        :term:`publications<publication>` are served.
+
+    :class:`~pulpcore.plugin.models.Importer`
+        User facing settings that specify how Pulp should to interact with an external
+        :term:`content` source.
+
+    plugin
+        A `Django <https://docs.djangoproject.com>`_ app that exends :term:`pulpcore` to manage one or more
+        :term:`types<type>` of :term:`content`.
+
+    publication
+        The metadata and :term:`artifacts<Artifact>` of the :term:`content units<content>` in a
+        :term:`repository version<RepositoryVersion>`. Publications are served by the
+        :term:`content app` when they are assigned to a :term:`distribution`.
```

Comment:
Just a thought here for clarity:  "The metadata and artifacts of the content units"--does the metadata include anything other than the content unit's type?  If so, we should perhaps list what metadata is in our definition of Content above.

Reply:
Ah thanks, this wording is unclear. I'll rework this section.

So you know whats up: The artifacts belong to each content unit, but the metadata I am referring to is a file that relates to the set of content. For instance, the rpm plugin uses an xml file that specifies which rpms are in the repository. When a version is published, pulp will write that xml file, and associate it to the publication.